src/main/play-doc/gettingStarted/Install.md

```diff
@@ -11,14 +12,22 @@ If you want to first get familiar with the terms and components of ConductR go t
 * [[ConductR components|ConductrComponents]]
 
 
+# Dev Installation
```

Comment:
I think that mixing in developer concerns into this section should be avoided. We should be mindful of the two audiences for our doc. This doc is particularly focused on ops.